provisioning/image_customization.go

```diff
@@ -235,7 +235,7 @@ func newImageCustomizationDeployment(info *ProvisioningInfo, ironicIP string) *a
 }
 
 func EnsureImageCustomizationDeployment(info *ProvisioningInfo) (updated bool, err error) {
-	ironicIP, err := GetIronicIP(info.Client, info.Namespace, &info.ProvConfig.Spec)
+	ironicIP, err := GetIronicIP(info.Client, info.Namespace, &info.ProvConfig.Spec, info)
```

Comment:
If you pass info, you no longer need to pass everything else. So we should either pass only OSClient or only info. I vote for the former because this way it will be more reusable by e.g. assisted services

Reply:
Good point, thank you Dmitry. I altered the patch to pass _info.OSClient_.